tests/components/samsungtv/test_config_flow.py

```diff
@@ -165,14 +166,6 @@
     },
     "id": "123",
 }
-MOCK_DEVICE_INFO_2 = {
-    "device": {
-        "type": "Samsung SmartTV",
-        "name": "fake2_name",
-        "modelName": "fake2_model",
-    },
-    "id": "345",
-}
```

Comment:
Small side cleanup - as the constant was unused (probably a left over from previous refactoring).